schema-extension.md

```diff
@@ -0,0 +1,162 @@
+# CAR schema extension
+
+The CAR schema can be extended by creating `get_schema_extension` method in `app.py` file of the connector and returning `SchemaExtension` object.
```

Comment:
The CAR schema can be extended by creating **a** `get_schema_extension` method in  **the** `app.py` file of the connector and returning `SchemaExtension` object.